lib/radius.js

```diff
@@ -449,6 +449,9 @@ Radius.decode_attributes = function(data, attr_hash, vendor, raw_attrs) {
     length = data.readUInt8(1);
     value = data.slice(2, length);
 
+    if (length < 2)
```

Comment:
Can you add curlies for the "if" body? We prefer that style since it is "safer" if someone needs to add more code in this "if" condition.